packages/fluentui/docs/src/examples/components/Toolbar/Visual/ToolbarExampleVariables.shorthand.tsx

```diff
@@ -67,6 +66,7 @@ const ToolbarExampleVariables = () => {
           icon: <MoreIcon />,
           key: 'item-menu',
           menu: {
+            accessibility: notAutoFocusToolbarMenuBehavior,
```

Comment:
These changes are related to the issue
https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/7148

Reply:
Thanks